package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "sinopia_user_backup",
+  "version": "0.0.1",
+  "description": "A Node component for backing up the Cognito user pool for Sinopia",
+  "bin": {
+    "backup": "bin/backup"
+  },
+  "scripts": {
+    "ci": "AWS_PROFILE=foobarbazquux jest --colors --ci --coverage",
```

Comment:
I don't love the env vars here either, but it works and I wasn't sure there was a better approach. Explained here: https://github.com/LD4P/sinopia_user_backup/pull/2/files#diff-5a61b7aef07cd51690e0d92206685355R13